src/js/components/Notification/propTypes.js

```diff
@@ -0,0 +1,13 @@
+import PropTypes from 'prop-types';
+
+let PropType = {};
+if (process.env.NODE_ENV !== 'production') {
+  PropType = {
+    title: PropTypes.string,
```

Comment:
see previous comment re: type definitions for required/optional should be synched.